tests/cache/tests.py

```diff
@@ -173,13 +180,21 @@ def test_unicode(self):
 
     def test_set_many(self):
         "set_many does nothing for the dummy cache backend"
-        cache.set_many({'a': 1, 'b': 2})
-        cache.set_many({'a': 1, 'b': 2}, timeout=2, version='1')
+        self.assertEqual(cache.set_many({'a': 1, 'b': 2}), [])
+        self.assertEqual(cache.set_many({'a': 1, 'b': 2}, timeout=2, version='1'), [])
```

Comment:
You've added some assertions to `test_set_many`.

I notice that `test_delete_many` and `test_clear` don't actually have any assertions. Perhaps you could add some? (Particularly for the former as that is changed in this PR.)

Reply:
There's nothing that can really be sensibly tested in those tests, except that they return `None`, which is the default for functions anyway?

Reply:
I see your point and can't think of anything sensible either.
Assertion-less tests just seem pointless other than to put a tick in the coverage box.

Reply:
They aren't pointless, we know the functions consume those arguments. This is a weird case because we *expect* the functions in question to do nothing, but there's of course no way to assert that :)